text/0000-offline-buildpackages.md

```diff
@@ -0,0 +1,234 @@
+# Meta
+[meta]: #meta
+- Name: Offline Buildpackages
+- Start Date: 2020-04-27
+- Author(s): Dan Thornton
+- RFC Pull Request: (leave blank)
+- CNB Pull Request: (leave blank)
+- CNB Issue: (leave blank)
+
+# Summary
+[summary]: #summary
+
+One paragraph explanation of the feature.
+
+We need to have a method to attach dependencies to buildpackages to support offline workflows. These dependencies should then be made available via the filesystem during the `build` phase.
+
+# Motivation
+[motivation]: #motivation
+
+- Why should we do this?
+    - Remove need for two separate `ids` for online vs offline buildpacks
```

Comment:
Would it be worth describing further how we'll preserve online vs. offline behavior? I assume something like, if a buildpack doesn't see its dependencies in the file system, go download them... or whatever is the intention.

Reply:
Yep, that was the initial idea, online vs offline behavior would require logic in each buildpack ( buildpack authors would be responsible). There was some discussion about trying to reduce this and provide a better experience for buildpack authors.